### ISSUE TITLE
token-transformer

### ISSUE THREAD

**Issue by obgarmo** [six7/figma-tokens#259]
Hi,
A question about the `token-transformer` package. This works quite well for me. Would you consider opening its source and allowing contributions?

**Comment by six7** [comment]
Definitely! I'd like to release it first to main branch though, it currently lives in the git sync branch which will be merged to main once git sync is out of beta (mid October)
